web/text_accessibility.js

```diff
@@ -31,6 +31,8 @@ class TextAccessibilityManager {
 
   #waitingElements = new Map();
 
+  #turnedOff = false;
+
```

Comment:
Similarly here, can't we tweak the existing `disable`-method rather than introduce this new functionality (possibly with and optional parameter for the `disable` method)?